action-server/covidflow/utils/testing_locations.py

```diff
@@ -133,7 +136,7 @@ async def _fetch_testing_locations_with_backoff(
 
 async def get_testing_locations(
     latitude: float, longitude: float
```

Comment:
maybe we should change that to `tuple[float, float]` to be more consistent with the rest...

Reply:
I think I'll do even better with a named tuple. I know the order is a convention, but it still annoys me that we don't know which one is which.

Reply:
:O I didn't even know named tuple existed